reframe/core/logging.py

```diff
@@ -427,6 +428,15 @@ def _update_check_extras(self):
         if self.check.job:
             self.extra['check_jobid'] = self.check.job.jobid
 
+        if self.check.job:
+            if self.check.job.completion_time:
+                # Format is the same for every handler
+                fmt = self.logger.handlers[0].formatter.datefmt
+                self.extra['check_completion_time'] = (
+                    self.check.job.completion_time.strftime(fmt))
```

Comment:
Try assigning the `self.check.job.completion_time` to a different variable to avoid the line split here.